src/wavm/mpi.cpp

```diff
@@ -594,11 +610,12 @@ WAVM_DEFINE_INTRINSIC_FUNCTION(env,
                                I32 requestPtrPtr,
                                I32 status)
 {
-    faabric::util::getLogger()->debug(
-      "S - MPI_Wait {} {}", requestPtrPtr, status);
-
     ContextWrapper ctx;
+
+    // ctx.getMpiLogger()->debug("S - MPI_Wait {} {}", requestPtrPtr, status);
```

Comment:
Can we lose this commented line in some future PR? Doesn't need to be done now

Reply:
Good catch, thanks.